custom_components/ha-google-home/sensor.py

```diff
@@ -46,11 +61,14 @@ def device_state_attributes(self):
 class GoogleHomeAlarmSensor(GoogleHomeAlarmEntity):
     """Representation of a Sensor."""
 
-    def __init__(self, coordinator, entry, device):
+    def __init__(self, coordinator, entry, device_name, device_data):
         """Initialize the sensor."""
         super().__init__(coordinator, entry)
-        self._name = device.device_name
-        self._alarms = [format_alarm_information(alarm) for alarm in device.alarms]
+        self._name = device_name
+        self._alarms_timestamp = device_data[LABEL_ALARMS]
+        self._alarms = [
+            format_alarm_information(alarm) for alarm in device_data[LABEL_ALARMS]
```

Comment:
`format_alarm_information` expects timestamp, so it should be `alarm["fire_time"]`